packages/wallets/src/features/cfd/CFDPlatformsList.scss

```diff
@@ -4,12 +4,40 @@
 
     &__cfd-empty-state {
         display: flex;
-        padding: 2.4rem 0rem 4.8rem;
-        flex-direction: column;
-        justify-content: center;
+        flex-direction: row;
         align-items: center;
-        gap: 2.4rem;
-        align-self: stretch;
+        justify-content: flex-start;
+        gap: 1.6rem;
+        height: 7.2rem;
```

Comment:
hey @lubega-deriv 
one concern that i would like to raise is fixed height for elements that have text inside.
definitely we can use fixed height in the code
but
be careful when we do it with the text.
I would like to suggest you to use vertical paddings instead of fixed text because let say in mobile view in EN there are two lines of text but in other language it can be 3, since you height is fixed, your text will overlap the element borders. hope you got my point 🙏😅
![Uploading Screenshot 2024-10-02 at 10.02.42.png…]()

Reply:
Updated